nodeJS/authentication/authentication_basics.md

```diff
@@ -48,6 +50,7 @@ const mongoose = require("mongoose");
 const Schema = mongoose.Schema;
 
 const mongoDb = "YOUR MONGO URL HERE";
+mongoose.set('strictQuery', false);
```

Comment:
Strict query is false by default. Is this line necessary?

Reply:
You are correct, will remove this.